src/pages/Login.tsx

```diff
@@ -0,0 +1,15 @@
+function Login() {
```

Comment:
Let's add Login and Register in features/auth folder
![image](https://github.com/user-attachments/assets/73676a6f-11b2-4931-9e22-e288c2e68b61)